app/relationfeaturesmodel.h

```diff
@@ -0,0 +1,53 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef RELATIONFEATURESMODEL_H
+#define RELATIONFEATURESMODEL_H
+
+#include "featureslistmodel.h"
```

Comment:
is this include required in header?

Reply:
Base class, did not work in forward declaration